src/psutil/psutil_linux.nim

```diff
@@ -166,13 +168,13 @@ proc pids*(): seq[int] =
     ## Returns a list of PIDs currently running on the system.
     let all_files = toSeq( walkDir(PROCFS_PATH, relative=true) )
 
-    return mapIt( filterIt( all_files, isnumder(it.path) ), parseInt( it.path ) )
+    return mapIt( filterIt( all_files, isnumber(it.path) ), parseInt( it.path ) )
 
 
 proc pid_exists*( pid: int ): bool =
     ## Check For the existence of a unix pid
 
-    let exists = psutil_posix.pid_exists( pid )
+    let exists = psutilim_posix.pid_exists( pid )
```

Comment:
Is this a typo?

Reply:
yes. I had them in plenty of files. I had them under different filenames to not get mixed up with hold and new. I just went to fix them